main.py

```
@@ -0,0 +1,35 @@
+        if is_open:
+            thread = threading.Thread(target=self.read_data)
+            thread.start()
+    
+    def read_data(self):
+        while True:
+            message = self.serial.readline()
+            print(message)
+
+
+if __name__ == "__main__":
+    peakCanSerial = PeakCanSerial()
```

Ruff (0.8.2)

Undefined name `PeakCanSerial`

(F821)